OracleCoherence/dockerfiles/14.1.1.0.0/src/main/docker/start.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env sh
+
+#!/bin/sh -e -x -u
+
+# Copyright 2019, 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Please change this header to `Copyright (c) 2019, 2020 Oracle and/or its affiliates.`